httpserver/service_node.cpp

```diff
@@ -108,47 +108,9 @@ constexpr std::chrono::milliseconds SWARM_UPDATE_INTERVAL = 1000ms;
 #endif
 constexpr std::chrono::seconds STATS_CLEANUP_INTERVAL = 60min;
 constexpr std::chrono::seconds OXEND_PING_INTERVAL = 30s;
-constexpr std::chrono::minutes POW_DIFFICULTY_UPDATE_INTERVAL = 10min;
 constexpr std::chrono::seconds VERSION_CHECK_INTERVAL = 10min;
 constexpr int CLIENT_RETRIEVE_MESSAGE_LIMIT = 10;
 
-static std::shared_ptr<request_t> make_push_all_request(std::string&& data) {
-    return build_post_request("/swarms/push_batch/v1", std::move(data));
```

Comment:
The handler for the endpoint in `http_connection.cpp` should also be removed; it seems nothing has called it in a long time, and this is removing the only place that referenced it.

Reply:
Updated